x-pack/plugins/fleet/common/types/index.ts

```diff
@@ -26,6 +26,7 @@ export interface FleetConfigType {
       host?: string;
       ca_sha256?: string;
     };
+    fleetServerUrls?: string[];
```

Comment:
do you think we still need the `kibana` setting now that we have this fleet server setting? I'd guess so based on the PR description of still supporting Kibana for now.

would it make sense to have this setting match the data structure of the kibana/ES settings? like
```
fleetServer?: {
  hosts?: string[];
  ca_sha256?: string;
}
```
I don't know if `ca_sha256` is applicable to fleet server or not

Reply:
there is no `ca_sha256` for fleet server for now, but I can change to have a similar structure if need to add more properties in the future 👍